src/domains/wallet/pages/WalletDetails/WalletDetails.tsx

```diff
@@ -65,8 +65,8 @@ export const WalletDetails = ({ wallet, wallets }: Props) => {
 	];
 
 	const handleDeleteWallet = async () => {
-		const wallet = activeProfile?.wallets().findById(walletId);
-		activeProfile?.wallets().forget(wallet?.id() as string);
+		const activeWallet = activeProfile?.wallets().findById(walletId);
```

Comment:
@faustbrian referring to the issue we were discussing, the problem before was caused by getting `activeWallet` in line [58](https://github.com/ArkEcosystem/desktop-wallet/pull/2534/commits/e2f798dabe10ccf73a8cb37a809b0de36fdd2a2d#diff-f3a761ca27f72fc8497491470b58c2d3R58).